vue/components/ui/molecules/avatar-list/avatar-list.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+    <div class="avatar-list">
+        <div
+            class="avatar-container"
+            v-bind:class="classes"
+            v-bind:style="{ 'z-index': index }"
+            v-for="(avatar, index) in avatarList"
+            v-bind:key="index"
+        >
+            <div v-if="index == maxElems">
+                + {{ avatarList.length - maxElems }}

```

Comment:
this should be moved to a prop

Reply:
A prop, or a computed variable? The text here is dynamic, since it depends on how many elements the list has, and the given `maxElems` prop.

Reply:
Sorry, I meant a computed